src/modules/navigator/mission.cpp

```diff
@@ -567,15 +599,19 @@ Mission::set_mission_items()
 		/* move to land wp as fixed wing */
 		if (_mission_item.nav_cmd == NAV_CMD_VTOL_LAND
```

Comment:
It takes precedence over the one below, and the one below kicks in too in the end. The sequence is long (to make sure no waypoint gets missed):
- vtol land
- move to land (as FW)
- transition
- move to land (as MC)